Vungle/src/main/java/com/mopub/mobileads/VungleRouter.java

```diff
@@ -278,7 +285,11 @@ public void updateConsentStatus(Vungle.Consent status) {
     private void clearWaitingList() {
         for (Map.Entry<AdRequest, VungleRouterListener> entry : sWaitingList.entrySet()) {
             AdRequest request = entry.getKey();
-            Vungle.loadAd(request.placementId, request.adMarkup, null, loadAdCallback);
+            if(AdSize.isNonMrecBannerAdSize(request.adSize)) {
+                Vungle.loadAd(request.placementId, request.adMarkup, null, loadAdCallback);
+            } else {
+                Banners.loadBanner(request.placementId, request.adMarkup, request.adSize, loadAdCallback);
+            }
```

Comment:
@palrahul sVungleRouterListeners below is based on "request.placementId"
So how it will resolve multiple requests for the same placementId but different AdRequest?
At least they will overwrite each other